lib/vagrant-ohai/action_configure_chef.rb

```diff
@@ -19,7 +19,7 @@ def call(env)
       private
       def ohai_custom_config(current_conf)
         tmp = Tempfile.new(["chef-custom-config", ".rb"])
-        tmp.puts 'Ohai::Config[:plugin_path] << "/etc/chef/ohai_plugins"'
+        tmp.puts 'ohai.plugin_path << "/etc/chef/ohai_plugins"'
```

Comment:
I believe the new approach is set out here:

https://github.com/chef-cookbooks/ohai/blob/master/libraries/plugin_helpers.rb#L62